### ISSUE TITLE
Hello for Business - Multi-factor Unlock - Login with Password only?

### ISSUE THREAD

**Issue by CHfish-ch** [MicrosoftDocs/windows-itpro-docs#10762]
Can you please share if a password-based login is still possible?
Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 38f8c2ce-8edb-60d3-ebf9-bdb625bf66dc
* Version Independent ID: ee1ed0a8-b05c-c499-6504-40b5b298c53b
* Content: [Multi-factor Unlock - Windows security](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/feature-multifactor-unlock)
* Content Source: [windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md)
* Product: **m365-security**
* GitHub Login: @Gitprakhar13
* Microsoft Alias: **prsriva**

**Comment by RAJU2529** [comment]
@CHfish-ch , yes possible, if you signed in with your local account on windows.

**Comment by CHfish-ch** [comment]
Thank you @RAJU2529 
I'm currently looking for a matrix / table of the different options with local, domain and Azure Account, especially if (how) one can could enforce MFA-Authentication to Windows with Hello for Business (in other words preventing a login with password only).
Do you have any information on that?

**Comment by RAJU2529** [comment]
@CHfish-ch . No other information sorry

**Comment by yogkumgit** [comment]
@Gitprakhar13 Please help us in resolving this issue. Thanks

**Comment by paolomatarazzo** [comment]
Hello @CHfish-ch,
password login is still a viable option, unless you disable the password credential provider or require the use of smart card.
we don't have a table with the different options. You can review the following documentation to go passwordless and essentially moving to MFA https://learn.microsoft.com/windows/security/identity-protection/hello-for-business/passwordless-strategy 

Hope this helps!
